readthedocs/projects/tasks.py

```diff
@@ -269,11 +272,13 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
                         'version': self.version.slug,
                     },
                 },
-
             )
         return False
 
 
+# Exceptions under ``throws`` argument are considered ERROR from a Build
+# perspective but as a WARNING for the application itself. These exception are
+# logged as ``INFO`` and they are not sent to Sentry.
```

Comment:
I don't understand this comment. "considered ERROR from a Build
 perspective but as a WARNING for the application itself" -- what does that mean in practice? Does the build stop?

Reply:
A known error "by RTD's code" like "we didn't find the `requirements.txt` you configured in the YAML file" is an ERROR that will make the build to stop. It's an error from the user's perspective.

Although, it's just a WARNING from the developer's perspective or the application (RTD's code) itself.

The build stops and the exception is logged as INFO/WARNING not as an ERROR.

If you have a better way to explain this we can adapt the comment. I'd appreciate.